### DISCUSSION TITLE
Error: exec: "ssh-keygen": executable file not found in %PATH%

### DISCUSSION THREAD

**Discussion by ismaelhamed** [podman-desktop/podman-desktop#6593] [General]
I'm trying to set up a Podman Desktop (`1.8.0`) in a W11 machine, but I'm getting the following error when creating the `podman-machine-default`:

```plain
Error: Command execution failed with exit code 125 Command execution failed with exit code 125 Error: exec: "ssh-keygen": executable file not found in %PATH%
```

The actual log output is:

```plain
Error: exec: "ssh-keygen": executable file not found in %PATH%

Command execution failed with exit code 125
Command execution failed with exit code 125
Error: exec: "ssh-keygen": executable file not found in %PATH%
```

ssh-keygen is in fact in the PATH (`C:\Program Files\OpenSSH\`) and I can run it with no problem:

```plain
PS> ssh-keygen
Generating public/private ed25519 key pair.
Enter file in which to save the key (C:\Users\<...>/.ssh/id_ed25519):
```

**Comment by ismaelhamed** [comment]
Obviously, restarting the laptop did the trick.

**Comment by rgaiacs** [comment]
I had the same problem. The OpenSSH client on Windows 11 is a optional feature.

![Screenshot of optional features dialog](https://github.com/user-attachments/assets/8dfc9f96-9d6b-42e5-8267-c440f55ab153)

Users need to navigate to `Settings`, `System`, `Optional features`.
